samples/client/petstore/clojure/pom.xml

```diff
@@ -1,32 +0,0 @@
-<project>
```

Comment:
@f-f please add back pom.xml so that CI (circleci) can run the tests for Clojure Petstore sample

Reply:
Yes sorry, I accidentally deleted them.

Reply:
No problem. Let us know if you need help running the tests locally for verification.